w3-account.md

```diff
@@ -0,0 +1,224 @@
+
+
+# Introduction
+
+## Motivation
+
+In web3.storage users MAY create number of user spaces simply by generating asymmetric key pair. They MAY be delegated capabilities for other user spaces. Managing these delegations and keypairs across multiple spaces, agents and devices can get complicated.
+
+To address this we propose a concept of an account, which is a [principal][] that can be delegated all relevant capabilities across various user spaces.
+
+We also propose account authorization flow that would allowing allow authorized agent to act on behalf of the account [principal][].
+
+
+# Terminology
+
+## Account
+
+User account is a [principal][] identified by [`did:mailto`][] (listed in a UCAN's `iss` or `aud` field).
+
+When user agent creates a new space, it MAY delegate full or subset of the capabilities to user account so they could be reclaimed by a user with another agent.
+
+> Example shows all capabilities to `did:key:zAlice` space been delegated to the user account `alice@web.mail` 
+
+```ts!
+{
+  iss: "did:key:zAlice",
+  aud: "did:mailto:alice@web.mail",
+  att: [{ with: "did:key:zAlice", can: "*" }],
+  exp: null,
+  sig: "..."
+}
+```
+
+## Delegations
+
+Agent MAY publish account delegations to a serivce so that they can be persisted and retrieved later with a different agent.
+
+> Invokes `access/delegate` asking web3.storage to record delegation from `did:key:zAlice` space to the `alice@web.mail` account.
+
+ ```ts
+{
+  iss: "did:key:zAgent",
+  aud: "did:dns:web3.storage",
+  att: [{
+    with: "did:key:zAgent",
+    can: "access/delegate",
+    nb: {
+      access: Link<{
+        iss: "did:key:zAlice",
+        aud: "did:mailto:alice@web.mail",
+        att: [{ with: "did:key:zAlice", can: "*" }],
+        exp: null,
+        sig: "..."
+      }>
+     }
+    }
+  }],
+  sig: "..."
+}
+```
+
+:::warning
+[Recipient validation][] requires wrapping actual delegation into `access/delegate` invocation, but in the future we may find a way to remove this requirement.
+:::
+ 
+## Authorization
+
+User MAY authorize an agent to represent their account by invoking `ucan/issue` capability with an audience where authorization is to be considered valid
+
+
+> Agent requests authorization to represent `alice@web.mail` with `did:key:zAgent` in the context of `web3.storage`
+
+```ts
+{
+  iss: "did:key:zAgent",
+  aud: "did:dns:web3.storage",
+  att: [{
+    with: "did:key:zAgent",
+    can: "ucan/issue",
+    nb: { as: "did:mailto:alice@web.mail" }
+  }]
+}
+```
+
+### issue `with`
+
+Resource MUST be a [`did:key`][] URI. It represents a public key that user wishes to use for signing UCANs issued by DID in the [`nb.as`][issue `as`] field.
+
+### issue `as`
+
+Account MUST be a [`did:mailto`][] principal. It is an account that user wishes to be represented by [`did:key`][] in the [`with`][issue `with`] field.
```

Comment:
See earlier comment. I don't think it's a good idea to have normative language that depends on a spec with no contents (`did:mailto`)